### PR TITLE
button on surface scene to leave now

### PR DESCRIPTION
**Pull request by Mokimbo**
changed some stuff in the planet scene
- changed the main node to a control node so we can manage the buttons to switch to different backgrounds potentially
- added a background that you see when you switch to it, thinking about making this background the one for refueling
- there are buttons now! the only one that works is the leave button which takes over what we used to press esc for. esc should now only open the game menu.

i also added the image i used for the backround to the general art folder under a planet surface backgrounds folder i added

### CODE REVIEW

**Comment by Mokimbo** [comment]
changed the input from "select_planet" to "event" and changed "Input." on 105 to "event.". Allows us to add more buttons for the ship in the same input function later. Landing should still work.

**Comment by Mokimbo** [comment]
Did a rebase with main to get the current save mechanic to help with testing the planet surface scene. It should now stretch and scale without showing the grey boxes on the edges. the button box should also keep its scale relative to the scene.

File: scenes/Ships/Ship_controls.gd [hunk 1]
```diff
@@ -101,8 +101,8 @@ func handle_landing_request():
 			print ("Landing sequence initiated")
 			GameState.scene = Constants.SceneId.PlanetSurface
 
-func _input(select_planet):
-	if Input.is_action_pressed("select_planet"):
+func _input(event):
```
Comment (alarner) [inline scenes/Ships/Ship_controls.gd:104]: Thanks for fixing this!

File: scenes/PlanetSurface/PlanetSurface.gd [hunk 2]
```diff
@@ -10,9 +10,6 @@ extends CanvasLayer
...
 	pass # Replace with function body.
 
-func _input(event) -> void:
-	if event.is_action_pressed("take_off"):
-		GameState.scene = Constants.SceneId.World
-# Called every frame. 'delta' is the elapsed time since the previous frame.
-#func _process(delta):
-#	pass
+
+func _on_Leave_pressed():
```
Comment (alarner) [inline scenes/PlanetSurface/PlanetSurface.gd:14]: Do we still want to support the take_off keyboard shortcut? If not we should remove it from the key mappings. If so we should keep the _input function as well. My vote would be to keep it, but maybe change it to the L key for "leave" (and it also matches L for land so you use the same key to change to and from the scene. What do you guys think?

**Review by alarner — APPROVED** [review]
🌮

**Pull request merged**